index.html

```diff
@@ -1,299 +1,318 @@
-                </nav>
-            </header>      
-
-            <section class="headline">            
-                <div class="headerh1">
-                <h1>I’m Samuel k.
-                    Glad to see you!</h1>
-                </div>
-                <div class="headerdetails">
-                    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-                    
-                </div>
-                <div class="letconnects">
-                <h4>LET’S CONNECT</h4>  
-                </div>
-                <div class="socialmediaclass">
-                    <img id="socialmedia" src="./img/socialmedia.png">
-                </div>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>portfolio</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <nav class="top-menu">
+        <div class="sub-menu">
+          <h1>Samuel Kinyanjui</h1>
+          <ul>
+            <li id="down"><a href="#">Portfolio</a></li>
+            <li id="down"><a href="#">About</a></li>
+            <li id="down"><a href="#">Contact</a></li>
+          </ul>
+          <img src="./image/hamburguer1111.png" alt="humber" id="humb">
+        </div>
+      </nav>
+      <section class="main-bar">
+      <header>
+        <div class="sub-main">
+          <h1>I’m Samuel <br>
+            Glad to see you!</h1>
+          <p>I’m a software developer! I can help you build a product , feature or 
+            website Look through some of my work and experience! If you like what you 
+            see and have a project you need coded, don’t hesitate to contact me.</p>
+            <div class="social">
+              <h4>LET’S CONNECT</h4>
+              <ul>
+                <li>
+                  <a href="#"
+                    ><img src="./image/twitter.png" alt="angellist"
+                  /></a>
+                </li>
+                <li>
+                  <a href="#"
+                    ><img src="./image/Linkedin icon.png" alt="Linkedin"
+                  /></a>
+                </li>
+                <li>
+                  <a href="#"><img src="./image/medium.png" alt="github" /></a>
+                </li>
+                <li>
+                  <a href="#"><img src="./image/github.png" alt="twitter" /></a>
+                </li>
+                <li>
+                  <a href="#"><img src="./image/angellist.png" alt="medium" /></a>
+                </li>
+              </ul>
+            </div>
+        </div>
 
-            </section>
-        </section>
+      </header>
     </section>
+<!--cards-->
+    <section class="works">
+      <nav class="card" id="one">
```

Comment:
- [ OPTIONAL ] I noticed that you wrapped your work cards in a `nav` tag and this is semantically not correct. The `<nav>` tag in HTML is typically used to represent a section of a page that contains navigation links. It is used to define a block of navigation elements that help users navigate through a website or a specific section of it. :+1: 
- The works cards, in this case, are not for navigation purposes so I will suggest that you remove the `nav` tags wrapping around the work cards and keep just the section tag :+1: 
- Please note that It's important to choose HTML tags that accurately represent the content they contain to ensure proper accessibility, maintainability, and understandability of your code. :100: